CHARTER.md

```diff
@@ -0,0 +1,196 @@
+
+Section Intentionally Left Blank
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+<!-- https://github.com/json-schema-org/community/discussions/286#discussioncomment-4391266 -->
+
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council (CPC). In the case of the JSON Schema project, it is delegated to the JSON Schema Technical Steering Committee ("TSC").
+
+### 2.1 Other Formal Project Relationships (optional)
+<!-- https://github.com/json-schema-org/community/discussions/286#discussioncomment-4391271 -->
+
+Section Intentionally Left Blank
+
+## Section 3: JSON Schema Org Governing Body (TSC)
+<!-- https://github.com/json-schema-org/community/discussions/286#discussioncomment-4391284 -->
+
+The TSC is initially established from the observed major contributors who are currently active and in good standing.
+
+There is no maximum TSC membership size. The TSC must have a minimum of four members.
+
+Changes to TSC membership should be posted in the agenda, and may be suggested as any other agenda item.
+
+TSC memberships are not time-limited.
+
+While the project is not looking to obtain "Impact" project status within the OpenJS Foundation, there is no requirement set out to limit the number of TSC members by employer. It is in some cases considered difficult or even unhelpful for the project to limit the number or percentage of TSC members by employer (Especially when an employer has employed individuals already active in the community to work exclusively on the open source project). While at this time there is no limits on TSC membership by employer, the TSC will strive to keep to at least less than 50%, ideally 33% (One third, one in three). The TSC will re-evaluate this specific clause at least every six months, and aim to revise it within one year to meet the "no more than 1/3 employer member affiliation" mandate.
+
+TSC members are expected to regularly participate in TSC activities.
+
+The TSC will meet regularly using virtual conferencing tools. The meeting will be directed by the TSC Chairperson(s). Responsibility for directing individual meetings may be delegated by a TSC Chairperson to any other TSC member. Minutes or an appropriate recording will be taken and made available to the community through accessible public postings.
+
+The TSC may, at its discretion, invite any number of non-voting observers to participate in the public portion of TSC discussions and meetings.
+
+A TSC member may be removed by vote from the TSC if, during a 3-month period, all of the following are true:
+
+- They attend fewer than 25% of the regularly scheduled meetings
+- They do not participate in any TSC votes
+- They do not provide any form of excuse or no excuse is known for their absence
+
+## Section 4: Roles & Responsibilities
```

Comment:
I'll suggest to include roles and responsabilities for Contributors and Committers:

Contributors include anyone in the community that contributes code, documentation, pull request reviews, or other technical artifacts to the Project (“Contributors”);
Committers are Contributors who have earned the ability to modify (“commit”) source code, documentation, or other technical artifacts in a project’s repository (“Committers”);
A Contributor may become a Committer of a given Project’s repository by majority approval of the repository Committers.
A Committer must be listed in the CODEOWNERS file of a given Project’s repository, along with the scope of their responsibility in the repository.
A Committer may be removed from a Project’s repository by the majority approval of the other Committers of the same repository.
Participation in the Project through becoming a Contributor and Committer is open to anyone so long as they abide by the terms of this Charter.

Reply:
I'm aware this aspect isn't defined, and I agree we should define it, however I'm not sure it belongs in the charter, but rather a separate Governance.md document.
We may also want to migrate some other content there, such as parts of the decision making process, when we have such a document ready to receive content.

I propose we don't define these roles now, but make it a task for the TSC.